unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Clipboard/Clipboard.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+
+public static class Clipboard
+{
+    static readonly Queue<ClipboardReadPromise> promises = new Queue<ClipboardReadPromise>();
+    private static readonly IClipboardImplementation impl = null;
+
+    public static void WriteText(string text)
+    {
+        impl?.RequestWriteText(text);
+    }
+
+    [Obsolete("Firefox not supported")]
+    public static ClipboardReadPromise ReadText()
+    {
+        ClipboardReadPromise promise = new ClipboardReadPromise();
+        promises.Enqueue(promise);
+        impl?.RequestGetText();
+        return promise;
+    }
+
+#if UNITY_WEBGL && !UNITY_EDITOR
+    public static void HookBrowserCopyPasteInput()
+    {
+        // NOTE: this does nothing but we'll use it to force the instantiation of this static class
+        // cause you need to call something for a static class to be instantiated.
+        // all the hooking is actually done inside ClipboardWebGL class.
+    }
+#endif
```

Comment:
Why not just create an instance?

Reply:
I think initializing or setting something up here will end in a unnecessary messy code when all we just need is for the static class to be created

Reply:
Here the code is relying on a language side-effect to get things ready, this is BAD. There has to be a better way to design this feature. If you can't come up with a better approach, let's talk this for a bit to see if we can rubber-duck a bit and improve it.